src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -892,7 +893,7 @@ public static String composeRegularCallableMethodHeaderSampleCode(
     return bodyStatements;
   }
 
-  private static List<Statement> composeUnaryCallableSampleCodeBodyStatements(
+  private static List<Statement> composeNonPagedCallableSampleCodeBodyStatements(
```

Comment:
Why not `UnaryCallable`? `NonPaged` may be more opaque to readers, since there are many different kinds of non-pagination RPCs.

Reply:
In `CallableMethodKind.REGULAR`, there 3 types: unary, paged, lro. 
Sample code in unary and LRO here are very similar, so I combine the body statements. And I will have two body statement methods: `composeNonPagedCallableSampleCodeBodyStatements`, `composePagedCallableSampleCodeBodyStatements`.

I have two options here:
1. split the `composeNonPagedCallableSampleCodeBodyStatements` into `composeUnaryCallableSampleCodeBodyStatements` and `composeLroCallableSampleCodeBodyStatements`. But the codes are very similar beside the `output type`.
2. Look for a better name for this method, maybe `composeRegularNonPagedCallableSampleCodeBodyStatements`, `composeUanryOrLroCallableSampleCodeBodyStatements` (🥺really bad on naming)

WDYT?

Reply:
Prefer `composeUnaryOrLroCallableSampleCodeBodyStatements` since it best conveys the method's semantics. 

Optional: if you'd like to shorten the method names, consider removing "SampleCode" from all the names since these are already in a class named `*SampleCodeComposer`.

Reply:
Sure, take `composeUnaryOrLroCallableSampleCodeBodyStatements`

I remove `SampleCode` from the private functions.